graphql/src/store/resolver.rs

```diff
@@ -230,11 +230,10 @@ impl StoreResolver {
                     // locate_block indicates that we do not have a block hash
                     // by setting the hash to `zero`
                     // See 7a7b9708-adb7-4fc2-acec-88680cb07ec1
-                    let hash_h256 = ptr.ptr.hash_as_h256();
-                    if hash_h256 == web3::types::H256::zero() {
+                    if ptr.ptr.hash.is_zero() {
                         None
                     } else {
-                        Some(r::Value::String(format!("0x{:x}", hash_h256)))
+                        Some(r::Value::String(format!("0x{:}", ptr.ptr.hash.hash_hex())))
```

Comment:
Probably that `BlockHash` should implement [`fmt::LowerHex`](https://doc.rust-lang.org/std/fmt/trait.LowerHex.html#)

Reply:
Good catch. It already implements `LoweHex` but `hash_hex` was being used everywhere. I went ahead and replaced all usages with formatting in a separate commit. Let me know if that was too much, I can revert to just handling this case.